modules/ding_user/ding_user.module

```diff
@@ -39,6 +39,15 @@ function ding_user_preprocess_html(&$variables) {
       drupal_add_http_header('Cache-Control', 'no-store, must-revalidate');
       drupal_add_http_header('Pragma', 'no-cache');
     }
+    elseif (count($args) == 1 || isset($args[1]) && in_array($args[1], array('login', 'password'))) {
+      // Redirect anonymous user to front page login. This is to prevent the
+      // library user from seeing /user, login and password page.
+      // Unless in maintenance_mode: There's no front page login when in maintenance_mode, and a redirect prevent login.
+      $enable_login_page = variable_get('ding_user_show_login_page', 0);
+      if ($enable_login_page != 1 && !variable_get('maintenance_mode', 0)) {
```

Comment:
https://inlead.atlassian.net/browse/EASYOPAC-842

Reply:
This should be removed/not merged.